disnake/types/threads.py

```diff
@@ -69,9 +69,18 @@ class Thread(_ThreadOptional):
     message_count: int
     rate_limit_per_user: int
     thread_metadata: ThreadMetadata
+    flags: int
+    applied_tags: SnowflakeList
```

Comment:
These are both optional, and `applied_tags` appears to be nullable as well (should also have a warning/TODO there)

Reply:
Flags was moved to optional in 1c23993bf854eeb66c4d41228c5e11c5095f763b, also, when is `applied_tags` nullable? From my testing it can be missing/empty list/list of snowflake